backend/README.md

```diff
@@ -17,5 +17,15 @@
 - Install `sqlx-cli` with `cargo install sqlx-cli`
 - To create a new migration: run `sqlx migrate add -r <migration-name>`
   - Make sure the migration name is useful and each migration is focused to one thing
+  - Current migrations 
+    - messages
+    - user
```

Comment:
```suggestion
```
We don't need to document this, this would grow to be so stinking long.  The Migrations should be adequately named to document themselves and work both forwards and backwards.